pa11yci.config.js

```diff
@@ -4,6 +4,8 @@
  * Note, this config is also used as the source of URLs to test for Lighthouse CI.
  */
 
+const getRelativeConfigURLs = require("./src/common-lib/urls/getRelativeConfigURLs");
```

Comment:
Maybe the file name could indicate that these are URLs to use with third-party test systems? I am terrible at names, but something like `getDeploymentTestUrls`